bundle/manifests/openshift-builds-operator.clusterserviceversion.yaml

```diff
@@ -59,7 +59,7 @@ metadata:
     operatorframework.io/arch.ppc64le: supported
     operatorframework.io/arch.s390x: supported
   name: openshift-builds-operator.v1.1.0
-  namespace: openshift-builds
+  namespace: placeholder
```

Comment:
If we are opting for `placeholder` instead of specifying particular namespace, won't we need to set the `operatorframework.io/suggested-namespace`annotation?

Reply:
No, we don't have to. OLM will set this value anyway when the `Subscription` object is created.

Per a discussion I had with @kevinrizza - this annotation is in fact meaningless for OLM. It only matters for things like the OpenShift Web Console that build on top of OLM.

Reply:
@adambkaplan Without the `operatorframework.io/suggested-namespace` annotation how are we going to install in the `openshift-builds` namespace.
The `Subscription` needs to be created in the `openshift-builds` namespace right? But we don't have access to create the namespace itself.

Reply:
Echoing the conversation we had in sprint planning.

OpenShift's web console has special permission to create namespaces with `openshift-*` prefixes (cluster admins on ROSA or other managed versions don't even have this permission). There's a bit of a Rube-Goldberg system that ties it all together:

1. Operators add the suggested namespace annotation to their CSV object.
2. OpenShift web console recommends the namespace in the UI on the install screens/pages. It includes an info prompt that says the namespace will be created if it isn't present.
3. Cluster admins can choose to use the suggested namespace, OR pick from a set of existing namespaces.
4. Once the install namespace is picked, the OpenShift web console creates a `Subscription` object in said namespace. OLM takes over from here, ultimately creating a CSV object for the operator _in the same namespace as the `Subscription`_.

For operator bundles, the `namespace` here is kinda meaningless. I could revert this if you think this will be a source of confusion for the team.